decidim-core/vendor/assets/javascripts/form_datepicker.js.es6

```diff
@@ -20,7 +20,7 @@
         leftArrow: '<<',
         rightArrow: '>>'
       }).on('changeDate', (ev) => {
-        $(ev.target).siblings('input').val(exports.moment.utc(ev.date));
+        $(ev.target).siblings('input').val(exports.moment(ev.date));
```

Comment:
> The JS code was assuming the date was in UTC, this is wrong since the users set the date times in their timezone.

Just saw this in the PR description. I agree UTC is wrong, but local time is also wrong. The date(time) should be local to the timezone where the participatory process is happening... right?